README.reproducible.md

```diff
@@ -0,0 +1,148 @@
+#Armory Reproducible Build Documentation
+
+##Gitian
+
+Gitian is the build system that Bitcoin Core uses for reproducible builds.
+Armory has adopted it for OS X and Windows reproducible builds. This guide
+assumes you are using KVM as the virtualization technology to make the VMs
+that Gitian uses to build in, but gitian-tools also supports LXC.
```

Comment:
Is it possible to include an explanation of how to use LXC? If not, don't worry. The only one we really need to document is the method we use to put out builds, which will presumably be KVM.